src/conf_macros.m4

```diff
@@ -35,12 +35,12 @@ AC_DEFUN([WITH_PLUGIN_PATH],
 AC_DEFUN([WITH_PID_PATH],
   [ AC_ARG_WITH([pid-path],
                 [AC_HELP_STRING([--with-pid-path=PATH],
-                                [Where to store pid files for the SSSD [/var/run]]
+                                [Where to store pid files for the SSSD [/var/lib/sss/]]
                                )
                 ]
                )
-    config_pidpath="\"VARDIR\"/run"
-    pidpath="${localstatedir}/run"
+    config_pidpath="\"SSS_STATEDIR\"/"
+    pidpath="${localstatedir}/lib/sss/"
```

Comment:
My understanding is that it only makes sense to put anything (including PID file) into '/run' if any external process is expected to read it there.
So my intention was to stress 'Type=notify' service type (i.e. systemd doesn't watch SSSD PID file, it is for internal use only)
Now having read https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s15.html I'm not so sure...
But statement "System programs that maintain transient UNIX-domain sockets must place them in this directory or an appropriate subdirectory" makes me wonder if entire /var/lib/sss/pipes and maybe /mc/ should go to /run as well?

Reply:
I agree about keeping the pid file in /run/, that is what I expect as standard placement for services. I am indifferent about /var/lib/sss/pipes and /mc/ moving

Reply:
Thinking more about it... Is PID file required at all?
IIRC, the only use is to check at startup if SSSD is already running.
This can be done iterating over '/proc', similar to https://github.com/SSSD/sssd/blob/master/src/util/find_uid.c#L251

Reply:
Anyway, I moved PID file location to '/run/sssd/' in the latest update.